spec/unit/array_transformations_spec.rb

```diff
@@ -78,7 +78,7 @@
 
       input = Array.new(138_706) { |i| { 'key' => i } }
 
-      expect { map[input] }.to_not raise_error(SystemStackError, %r{stack level too deep})
+      expect { map[input] }.to_not raise_error
```

Comment:
```
WARNING: Using `expect { }.not_to raise_error(SpecificErrorClass, message)` risks false positives, since literally any other error would cause the expectation to pass, including those raised by Ruby (e.g. NoMethodError, NameError and ArgumentError), meaning the code you are intending to test may not even get reached. Instead consider using `expect {}.not_to raise_error` or `expect { }.to raise_error(DifferentSpecificErrorClass)`. This message can be suppressed by setting: `RSpec::Expectations.configuration.on_potential_false_positives = :nothing`. Called from /Users/kukunin/Sites/transproc/spec/unit/array_transformations_spec.rb:81:in `block (3 levels) in <top (required)>
```